homepage.md

```diff
@@ -26,7 +26,7 @@ Statistics about stored and transferred data:
 - **storage_min_healthy_pieces_count** - minimum number of healthy pieces per segment stored on storage nodes
 - **storage_remote_bytes** - number of bytes stored on storage nodes (does not take into account the expansion factor of erasure encoding)
 - **storage_remote_segments** - number of segments stored on storage nodes
-- **storage_remote_segments_lost** - number of irreparable segments lost from storage nodes
+- **storage_remote_segments_lost** - number of segments that the repairer could not download at moment, usually due to network problems.
```

Comment:
To be even more clear, we might say something like "usually due to network problems on the side of the repairer itself" because generic network problems on the node side is not supposed to affect our availability.